notes-app.js

```diff
@@ -7,7 +7,7 @@ const filters = {
 
 renderNotes(notes, filters)
 
-document.querySelector('#create-note').addEventListener('click', function (e) {
+document.querySelector('#create-note').addEventListener('click', (e) => {
```

Comment:
Minor: But since this function is taking only 1 argument, you could remove the parenthesis.
Same for all other arrow functions with just 1 argument.

Reply:
Done.